### DISCUSSION TITLE
Kade engine incorrect building

### DISCUSSION THREAD

**Discussion by Lowhere0** [KadeArchive/Kade-Engine#1667] [Q&A]
I want build a kade engine. I download all stuff. and make how in tutorial, but if i enter a "lime test windows" in folder of a code nothing is happen. I try in a Visual Code Studio, but received this:
export/release/html5/haxe/ManifestResources.hx:1: character 1 : Malformed file. Source files must be encoded with UTF-8.
export/release/html5/haxe/ApplicationMain.hx:13: lines 13-295 : Defined in this class
export/release/html5/haxe/ManifestResources.hx:1: character 1 : Malformed file. Source files must be encoded with UTF-8.
export/release/html5/haxe/ApplicationMain.hx:13: lines 13-295 : Defined in this class
export/release/html5/haxe/ManifestResources.hx:1: character 1 : Malformed file. Source files must be encoded with UTF-8.
export/release/html5/haxe/ApplicationMain.hx:13: lines 13-295 : Defined in this class
C:/Users//flixel/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/Users//flixel/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
The terminal process "C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe -Command lime build html5 " terminated with exit code: 1.

**Comment by isakube** [comment]
what the fff

**Reply by Lowhere0** [reply]
KadeEngine

**Reply by Lowhere0** [reply]
With no mods
